docs/how-to-guides/openstack_swift.md

```diff
@@ -0,0 +1,22 @@
+# Swift Storage How-To
+
+In order to use this charm along with Openstack swift storage you'll need to do the following:
+
+First, connect to your openstack instance with your credentials, normally sourcing an .rc file:
+```
+source myfile.rc
+```
+Then execute:
+```
+openstack object list default -vv
+```
+and you will see a debug log. THe URL should be located in the URL section of a curl command or similar. The swift URL looks like this:
+```
+http://10.126.72.107:8080/v1/AUTH_fa8326b9fd4f405fb1c5eaafe988f5fd/default
+```
+After obtaining the URL, configure the charm to use that url as backend and site:
```

Comment:
After obtaining the URL, configure the charm to use that url as backend and site. Assuming the charm has been deployed as `content-cache-k8s`, run: